diff -urt facter-3.1.4-orig/CMakeLists.txt facter-3.1.4/CMakeLists.txt
--- facter-3.1.4-orig/CMakeLists.txt	2016-01-21 21:14:37.000000000 +0100
+++ facter-3.1.4/CMakeLists.txt	2016-02-07 20:19:34.125448307 +0100
@@ -44,7 +44,7 @@
 include(options)
 
 # We use program_options, system, filesystem, date_time, and regex directly.
-find_package(Boost 1.54 REQUIRED COMPONENTS program_options system filesystem date_time regex)
+find_package(Boost 1.53 REQUIRED COMPONENTS program_options system filesystem date_time regex)
 
 find_package(Ruby 1.9)
 
diff -urt facter-3.1.4-orig/README.md facter-3.1.4/README.md
--- facter-3.1.4-orig/README.md	2016-01-21 21:14:37.000000000 +0100
+++ facter-3.1.4/README.md	2016-02-07 20:19:34.123448221 +0100
@@ -15,7 +15,7 @@
 
 * GCC 4.8+ or Clang 5.0+ (OSX)
 * CMake >= 3.2.2
-* Boost C++ Libraries >= 1.54
+* Boost C++ Libraries >= 1.53
 * yaml-cpp >= 0.5.1
 
 Currently the leatherman library is vendored as a git submodule.  To ensure submodules are initialized, please run:
@@ -94,9 +94,9 @@
     $env:PATH = "C:\tools\mingw64\bin;$env:PATH"
     $install = "C:\tools"
 
-    (New-Object Net.WebClient).DownloadFile("https://downloads.sourceforge.net/boost/boost_1_54_0.7z", "$pwd/boost_1_54_0.7z")
-    7za x boost_1_54_0.7z
-    pushd boost_1_54_0
+    (New-Object Net.WebClient).DownloadFile("https://downloads.sourceforge.net/boost/boost_1.53_0.7z", "$pwd/boost_1_54_0.7z")
+    7za x boost_1.53_0.7z
+    pushd boost_1.53_0
     .\bootstrap mingw
     .\b2 toolset=gcc --build-type=minimal install --prefix=$install --with-program_options --with-system --with-filesystem --with-date_time --with-thread --with-regex --with-log --with-locale --with-chrono boost.locale.iconv=off
     popd
diff -urt facter-3.1.4-orig/vendor/leatherman/file_util/CMakeLists.txt facter-3.1.4/vendor/leatherman/file_util/CMakeLists.txt
--- facter-3.1.4-orig/vendor/leatherman/file_util/CMakeLists.txt	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/file_util/CMakeLists.txt	2016-02-07 20:19:34.124448263 +0100
@@ -1,4 +1,4 @@
-find_package(Boost 1.54 REQUIRED COMPONENTS filesystem system)
+find_package(Boost 1.53 REQUIRED COMPONENTS filesystem system)
 
 add_leatherman_deps("${Boost_LIBRARIES}")
 add_leatherman_includes("${Boost_INCLUDE_DIRS}")
diff -urt facter-3.1.4-orig/vendor/leatherman/locale/CMakeLists.txt facter-3.1.4/vendor/leatherman/locale/CMakeLists.txt
--- facter-3.1.4-orig/vendor/leatherman/locale/CMakeLists.txt	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/locale/CMakeLists.txt	2016-02-07 20:19:34.124448263 +0100
@@ -1,9 +1,9 @@
 if(WIN32)
     set(LEATHERMAN_LOCALE_SRCS "src/windows/locale.cc")
-    find_package(Boost 1.54 REQUIRED COMPONENTS locale system)
+    find_package(Boost 1.53 REQUIRED COMPONENTS locale system)
 else()
     set(LEATHERMAN_LOCALE_SRCS "src/posix/locale.cc")
-    find_package(Boost 1.54 REQUIRED)
+    find_package(Boost 1.53 REQUIRED)
 endif()
 
 
diff -urt facter-3.1.4-orig/vendor/leatherman/logging/CMakeLists.txt facter-3.1.4/vendor/leatherman/logging/CMakeLists.txt
--- facter-3.1.4-orig/vendor/leatherman/logging/CMakeLists.txt	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/logging/CMakeLists.txt	2016-02-07 20:19:34.124448263 +0100
@@ -1,4 +1,4 @@
-find_package(Boost 1.54 REQUIRED COMPONENTS log log_setup thread date_time filesystem system chrono regex)
+find_package(Boost 1.53 REQUIRED COMPONENTS thread date_time filesystem system chrono regex)
 find_package(Threads)
 
 add_leatherman_deps(${Boost_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
diff -urt facter-3.1.4-orig/vendor/leatherman/nowide/CMakeLists.txt facter-3.1.4/vendor/leatherman/nowide/CMakeLists.txt
--- facter-3.1.4-orig/vendor/leatherman/nowide/CMakeLists.txt	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/nowide/CMakeLists.txt	2016-02-07 20:19:34.124448263 +0100
@@ -1,4 +1,4 @@
-find_package(Boost 1.54 REQUIRED)
+find_package(Boost 1.53 REQUIRED)
 
 add_leatherman_includes(${Boost_INCLUDE_DIRS} "${CMAKE_CURRENT_SOURCE_DIR}/../vendor/boost-nowide")
 add_leatherman_headers(../vendor/boost-nowide/boost)
diff -urt facter-3.1.4-orig/vendor/leatherman/README.md facter-3.1.4/vendor/leatherman/README.md
--- facter-3.1.4-orig/vendor/leatherman/README.md	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/README.md	2016-02-07 20:19:34.124448263 +0100
@@ -13,7 +13,7 @@
 
 ### Dependencies
 
-* Boost, at least version 1.54
+* Boost, at least version 1.53
 
 ### As a Standalone Library
 
diff -urt facter-3.1.4-orig/vendor/leatherman/util/CMakeLists.txt facter-3.1.4/vendor/leatherman/util/CMakeLists.txt
--- facter-3.1.4-orig/vendor/leatherman/util/CMakeLists.txt	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/util/CMakeLists.txt	2016-02-07 20:19:34.125448307 +0100
@@ -1,4 +1,4 @@
-find_package(Boost 1.54 REQUIRED date_time chrono system)
+find_package(Boost 1.53 REQUIRED date_time chrono system)
 
 add_leatherman_deps(${Boost_LIBRARIES})
 add_leatherman_includes("${Boost_INCLUDE_DIRS}")
diff -urt facter-3.1.4-orig/vendor/leatherman/windows/CMakeLists.txt facter-3.1.4/vendor/leatherman/windows/CMakeLists.txt
--- facter-3.1.4-orig/vendor/leatherman/windows/CMakeLists.txt	2016-01-21 21:14:41.000000000 +0100
+++ facter-3.1.4/vendor/leatherman/windows/CMakeLists.txt	2016-02-07 20:19:34.124448263 +0100
@@ -1,4 +1,4 @@
-find_package(Boost 1.54 REQUIRED)
+find_package(Boost 1.53 REQUIRED)
 
 add_leatherman_includes("${Boost_INCLUDE_DIRS}")
 
