diff -up facter-1.5.4/install.rb.libperms facter-1.5.4/install.rb
--- facter-1.5.4/install.rb.libperms	2008-09-08 23:00:03.000000000 -0400
+++ facter-1.5.4/install.rb	2009-02-28 14:15:53.000000000 -0500
@@ -93,7 +93,7 @@ def do_libs(libs, strip = 'lib/')
     op = File.dirname(olf)
     File.makedirs(op, true)
     File.chmod(0755, op)
-    File.install(lf, olf, 0755, true)
+    File.install(lf, olf, 0644, true)
   end
 end
 
